cl/api/templates/financial-disclosure-api-docs-vlatest.html

```diff
@@ -64,6 +66,8 @@ <h1 id="about">Financial Disclosures&nbsp;API</h1>
   </p>
   <p>Judicial officers and certain judicial employees in the United States are required to file financial disclosure reports by <a href="https://www.law.cornell.edu/uscode/text/5a/compiledact-95-521/title-I">Title I of the Ethics in Government Act of 1978</a>. The Act requires that designated federal officials publicly disclose their personal financial interests to ensure confidence in the integrity of the federal government by demonstrating that they are able to carry out their duties without compromising the public trust.
   </p>
+  <p>These APIs were used by the Wall Street Journal in <a href="https://www.wsj.com/articles/131-federal-judges-broke-the-law-by-hearing-cases-where-they-had-a-financial-interest-11632834421">their 17-part exposé</a> about the hidden conflicts of federal judges. That led to Congress passing a law to put this information online. It was also used by ProPublica in <a href="https://www.propublica.org/article/clarence-thomas-scotus-undisclosed-luxury-travel-gifts-crow">their Pulitzer prize winning reporting</a> about failures to disclose gifts and perks.
```

Comment:
how about this 

<p>These APIs were used by the Wall Street Journal in <a href="https://www.wsj.com/articles/131-federal-judges-broke-the-law-by-hearing-cases-where-they-had-a-financial-interest-11632834421">their 17-part exposé</a> about the hidden conflicts of federal judges. That led to Congress passing the <a href="https://www.govinfo.gov/content/pkg/PLAW-117publ125/pdf/PLAW-117publ125.pdf">Courthouse Ethics and Transparency Act</a> to put this information online. It was also used by ProPublica in <a href="https://www.propublica.org/article/clarence-thomas-scotus-undisclosed-luxury-travel-gifts-crow">their Pulitzer prize winning reporting</a> about failures to disclose gifts and perks.</p>

Reply:
Nice. Done.